react/lib/util/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const FIAT_CURRENCIES = ['USD', 'CAD', 'EUR', 'GBP', 'AUD'];
+
+export const CRYPTO_CURRENCIES = ['BCH', 'XEC'];
+
+export const DECIMALS: { [key: string]: number } = {
+    BCH: 8,
+    XEC: 8,
```

Comment:
Why isn't XEC = 2 here?

Reply:
nice catch, just fixed it and added tests to that file to make sure we don't miss it next time